src/prefabs/form.tsx

```diff
@@ -63,206 +71,297 @@ const beforeCreate = ({
   return (
     <>
       <Header onClose={close} title="Configure form" />
+      <Box
+        as="header"
+        direction="row"
+        justify="between"
+        align="center"
+        margin="1.5rem 2rem"
+      >
+        <Text margin={{ bottom: 'medium' }} size="0.875rem" color="grey700">
+          You can set up a form based on a model and selected properties so that
+          your form is created with matching inputs.
+          <br />
+          You can turn off the switch to not base your form on a model within
+          your app and manually configure it.
+        </Text>
+      </Box>
       <Content>
-        <Field
-          label="Select model"
-          error={
-            validationMessage && (
-              <Text color="#e82600">{validationMessage}</Text>
-            )
-          }
-        >
-          <ModelSelector
-            onChange={(id) => {
-              setModelId(id);
-            }}
-            margin
-          />
-        </Field>
-        <Field label="Select properties">
-          <PropertiesSelector
-            allowRelations
-            disabledKinds={[]}
-            disabledNames={['created_at', 'id', 'updated_at']}
-            modelId={modelId}
-            onChange={setProperties}
-            scopedModels={false}
-            value={properties}
-          />
+        <Field label="Model based form">
+          {/* TODO: fix onChange bug and move setmodelBased to Toggle onChange */}
```

Comment:
Do we need this TODO?

Reply:
yes, because this is a work around and doesnt suppose to work like this (onChange should be on Toogle)